cardano-explorer-db/src/Explorer/DB/Query.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
```

Comment:
what is `RankNTypes` doing in this context?

Reply:
It was needed for a something I was trying and then removed.

Have now removed it.